drivers/ethernet/Kconfig

```
@@ -32,6 +32,18 @@ config ETH_QEMU_IFACE_NAME
 	  script from net-tools project can be used to create the network
 	  interface.
 
+config ETH_QEMU_EXTRA_ARGS
+	string "Extra arguments to QEMU -nic option"
+	depends on NET_QEMU_ETHERNET
+	default ""
+	help
+	  Extra arguments passed to QEMU -nic option when Ethernet Networking
+	  is enabled. Typically this is used to set the network MAC address of
+	  Zephyr instance. This option can contain multiple QEMU option
+	  arguments. Each QEMU argument must be separated by comma ","
```

```suggestion
 	  arguments. Each QEMU argument must be separated by comma "," and no spaces
```

Have experienced people before thinking that they can and spaces in such cases, and write:
`arg0, arg1` which will clearly break in this situation.